### DISCUSSION TITLE
highs_interface works or not in v9.5?

### DISCUSSION THREAD

**Discussion by C8504** [google/or-tools#3626] [Linear Solver questions] [answered]
Dear experts,

Could I know HIGHS solver works or not in v9.5? I tried CBC SCIP, they work well, while HIGHS doesn't.

Thanks.

John

**Comment by lperron** [comment, marked as answer]
Not yet.

**Reply by C8504** [reply]
Thanks, could I know any plan please?

John

**Comment by lperron** [comment]
We are working with the Highs team to finish the integration.

We are dependent on their availability.

**Reply by C8504** [reply]
Thanks a lot for reply.

John
